src/algebra/free_algebra.lean

```diff
@@ -334,6 +334,19 @@ lemma algebra_map_left_inverse :
   function.left_inverse algebra_map_inv (algebra_map R $ free_algebra R X) :=
 λ x, by simp [algebra_map_inv]
 
+variables (R) (X)
+
+/-- The algebra map `R → (free_algebra R X)` is injective. -/
+lemma algebra_map_injective:
+  function.injective (algebra_map R (free_algebra R X)) :=
+begin
+  apply function.has_left_inverse.injective,
+  use algebra_map_inv,
+  exact algebra_map_left_inverse,
+end
+
+variables {R} {X}
```

Comment:
This lemma seems pretty pointless to me, you can get it already with `algebra_map_left_inverse.injective`

Reply:
you are right, my bad.